<launch> 
    <!-- Blood Pressure Measurement sensor-->    
    <node name="g3t1_4" pkg="component" type="g3t1_4" output="screen" />

    <param name="frequency" value="1" /> <!-- 1 Hz  -->

    <!-- Defines the percentages to consider low, moderate or high risk -->
    <param name="lowrisk" value="0,20" />
    <param name="midrisk" value="21,65" />
    <param name="highrisk" value="66,100" />

    <!-- markov chain for diastolic -->
    <param name="diasstate0" value="0,0,0,0,0" />
    <param name="diasstate1" value="0,0,0,0,0" />
    <param name="diasstate2" value="0,0,94,4,2" />
    <param name="diasstate3" value="0,0,50,35,15" />
    <param name="diasstate4" value="0,0,31,25,44" />

    <!-- risk values for diastolic -->
    <param name="diasLowRisk" value="0,80" />
    <param name="diasMidRisk" value="80,90" />
    <param name="diasHighRisk" value="90,300" />

    <!-- accuracy in percentage for diastolic -->
    <param name="diasdata_accuracy" value="90" type="double" />
    <param name="diascomm_accuracy" value="100" type="double" />

    <!-- accuracy in percentage for systolic -->
    <param name="systdata_accuracy" value="90" type="double" />
    <param name="systcomm_accuracy" value="100" type="double" />

</launch>
